src/core/canonical_constructor.jl

```diff
@@ -1,40 +1,81 @@
 
-function _canonical_model_init(bus_count::Int64,
+function _container_spec(V::DataType, ax...; kwargs...)
+
+    if :parameters in keys(kwargs)
```

Comment:
Suggest using this one-liner in all locations:
parameters = get(kwargs, :parameters, true)